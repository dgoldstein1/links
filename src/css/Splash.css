.App {
  height: 100%;
  overflow-y: hidden;
  max-height: stretch;
  align-items: center; 
  justify-content: center;
  text-align: center;
  background: linear-gradient(to bottom, orange, transparent 90%);
}

.App-header {
  height: 200px;
  padding: 100px;
}

.App-title {
  font-size: 10em;
}

.App-intro {
  font-style: italic;
  font-size: large;
}