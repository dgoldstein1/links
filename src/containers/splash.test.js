import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import Splash from "./splash";

const mockStore = configureStore([]);

describe.skip("Splash", () => {
  const defaultState = {};
  describe("matches snapshots", () => {
    let testTable = [
      {
        name: "cluster",
        state: defaultState,
      },
    ];

    testTable.forEach((t) => {
      it(t.name, () => {
        let store = mockStore(t.state);
        let wrapper = renderer.create(
          <Provider store={store}>
            <Splash />
          </Provider>
        );
        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});
