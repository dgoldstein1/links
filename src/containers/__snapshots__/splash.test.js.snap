// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash matches snapshots cluster 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Links
    </h1>
    <p
      className="App-intro"
    >
      Make Connections!
    </p>
  </div>
  <div
    className="contentContainer"
  >
    <div
      style={
        Object {
          "height": "400px",
          "width": "1024px",
        }
      }
    />
  </div>
  <footer
    className="App-footer"
  >
    Â© 2019 David Goldstein |
     
    <a
      href="http://davidcharlesgoldstein.com?ref=links-ui"
    >
      Personal Website
    </a>
     
    | 
    <a
      href="/LICENSE"
    >
      License
    </a>
     | 
    <a
      href="/VERSION"
    >
      Version
    </a>
  </footer>
</div>
`;
